stable/yugabyte/templates/_helpers.tpl

```diff
@@ -243,7 +243,7 @@ we stick to 0.0.0.0, which works for master.
 */}}
 {{- define "yugabyte.rpc_bind_address" -}}
   {{- $port := index .Service.ports "tcp-rpc-port" -}}
-  {{- if .Values.istioCompatibility.enabled -}}
+  {{- if (or .Values.istioCompatibility.enabled .Values.multicluster.createServicePerPod) -}}
```

Comment:
It would be better to set it to `$(POD_IP):{{ $port }}` i.e. same as createServiceExports. 
That seems to be a overall better option than 0.0.0.0.

Reply:
done @bhavin192